includes/polling/ports.inc.php

```diff
@@ -744,10 +744,14 @@
         $fields['ifOutErrors_rate'] = $port['ifOutErrors_rate'];
         $fields['ifInOctets_rate'] = $port['ifInOctets_rate'];
         $fields['ifOutOctets_rate'] = $port['ifOutOctets_rate'];
+        $fields['ifInBits'] = $port['stats']['ifInBits_rate'];
```

Comment:
you can't do this with opentsdb I have already searched with InfluxDB it's right you can do this  
![op2](https://user-images.githubusercontent.com/28858069/28109027-e5dd9d16-66ed-11e7-82f4-447de7537a88.PNG)

![gm](https://user-images.githubusercontent.com/28858069/28109035-edfb92f0-66ed-11e7-9aa0-d7dcce5ee28e.PNG)